eng/pipelines/installer/helix-queues-setup.yml

```diff
@@ -45,7 +45,10 @@ jobs:
 
     # OSX arm64
     - ${{ if eq(parameters.platform, 'osx_arm64') }}:
-      - osx.13.Arm64.Open
+      - ${{ if eq(parameters.jobParameters.isExtraPlatformsBuild, true) }}:
+        - OSX.26.Arm64.Open
+      - ${{ if ne(parameters.jobParameters.isExtraPlatformsBuild, true) }}:
```

Comment:
You can use `${{ else }}:` in this context

Reply:
That's what I thought, but everywhere else that it seems possible uses this pattern (which presumably is why GHCP chose to write it this way).